admin/class-yoast-form.php

```diff
@@ -404,25 +404,28 @@ public function hidden( $var, $id = '' ) {
 	/**
 	 * Create a Select Box.
 	 *
-	 * @param string $var    The variable within the option to create the select for.
-	 * @param string $label  The label to show for the variable.
-	 * @param array  $values The select options to choose from.
+	 * @param string $field_name     The variable within the option to create the select for.
+	 * @param string $label          The label to show for the variable.
+	 * @param array  $select_options The select options to choose from.
 	 */
-	public function select( $var, $label, $values ) {
-		if ( ! is_array( $values ) || $values === array() ) {
+	public function select( $field_name, $label, $select_options ) {
+
+		if ( ! is_array( $select_options ) || $select_options === array() ) {
```

Comment:
Why not use `empty( $select_options )` instead of doing a strict comparison to an empty array?